exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,17 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80) {
+    return "A";
+  } else if (mark < 80 && mark > 60) {
+    return "B";
+  } else if (mark <= 60 && mark >= 50) {
+    return "C";
+  }
+
+  return "F";
+}
```

Comment:
In this case, you can actually just do:
```
function calculateGrade(mark) {
  if (mark >= 80) {
    return "A";
  } else if (mark >= 60) {
    return "B";
  } else if (mark >= 50) {
    return "C";
  }

  return "F";
}
```
You don't need to check `mark < 80` in the second if statement because if it has reached that line, we definiely know that it is less than 80 or it would have passed in the first if statement. So in the second if statement, we only need to check `mark > 60`. I hope this makes sense?